tox.ini

```diff
@@ -183,8 +183,9 @@ commands_pre =
   psycopg2: pip install {toxinidir}/ext/opentelemetry-ext-dbapi
   psycopg2: pip install {toxinidir}/ext/opentelemetry-ext-psycopg2
 
+  pymysql: pip install {toxinidir}/opentelemetry-auto-instrumentation
   pymysql: pip install {toxinidir}/ext/opentelemetry-ext-dbapi
-  pymysql: pip install {toxinidir}/ext/opentelemetry-ext-pymysql
+  pymysql: pip install {toxinidir}/ext/opentelemetry-ext-pymysql[test]
```

Comment:
Is the [test] for using TestBase? How does this work?

Reply:
It's a way to tell pip to install opentelemetry-ext-pymysql **and** the dependencies on the extra test section: https://github.com/open-telemetry/opentelemetry-python/pull/611/files#diff-3874eac15d9a093a3db07c9283d97d12R48